tests/accordion-test.ts

```diff
@@ -0,0 +1,605 @@
+
+        containerElement.forEach((_,index)=>{
+            if(index === 1 || index === 3) {
+                expect(getComputedStyle(bodysel[index]).display).to.equal("block");
+            }
+        })
+        
+        expect(el.querySelectorAll(".acc-active").length).equal(2);
+        titles[1].click();
+        await el.updateComplete;
+        titles[3].click();
+        await el.updateComplete;
+        el.setAttribute("nomultiple", "");
+        await el.updateComplete;
+
+        titles[2].click();
+        await el.updateComplete;
+        titles[1].click();
+        await el.updateComplete;
+
+        expect(el.querySelectorAll(".acc-active").length).equal(1);
+        expect(el.hasAttribute("nomultiple")).to.be.true;
+        const activeItemsAfter = el.querySelectorAll(".acc-active");
+
+        console.log(el);
+        activeItemsAfter.forEach((item) => {
+            expect(
+                getComputedStyle(item.querySelector(".acc-body")).display
+            ).to.equal("block");
+        });
+
+        containerElement.forEach((_,index)=>{
+            if(index !== 4){
+                expect(getComputedStyle(bodysel[index]).display).to.equal("block")
+            }
+            else {
+                expect(getComputedStyle(bodysel[index]).display).to.equal("none");       
+            }
+        })
+    });
+
+    it("allows multiple items to be open if nocloseall is true then set as false", async () => {
+        const el = await fixture(
+            createAccordionTemplate(
+                { activecls: "acc-active", nocloseall: true },
+                [
+                    {
+                        title: "What is Artificial Intelligence?",
+                        body: "body 1",
+                        active: true,
+                    },
+                    {
+                        title: "What is Machine Learning?",
+                        body: "body 2",
+                        active: false,
+                    },
+                    { title: "title 3?", body: "body 3", active: true },
+                    { title: "title 4?", body: "body 4", active: false },
+                    { title: "title 5?", body: "body 5", active: true },
+                ]
+            )
+        );
+        const titles = el.querySelectorAll(".acc-title");
+        titles[0].click();
+        await el.updateComplete;
+        titles[1].click();
+        await el.updateComplete;
+        titles[2].click();
+        await el.updateComplete;
+        expect(el.querySelectorAll(".acc-active").length).to.equal(2);
+
+        const activeItems = el.querySelectorAll(".acc-active");
+        const containerElement = el.querySelectorAll(".container");
+        const bodysel = el.querySelectorAll(".acc-body");
+
+        containerElement.forEach((_,index) =>{
+            if(index == 0 || index == 1 || index == 2) {
+                expect(getComputedStyle(bodysel[index]).display).to.equal("block");
+            }
+        })
```

Comment:
This seems to be wrong. You are checking the active class to be of length  == 2 and in here you are checking block to be set for three items. Also 0th index was default open, and clicking on it should have closed it, but you are check it to be block in here. Same with the 2nd index.